ad-joining/register-computer/join.ps1

```diff
@@ -27,13 +27,61 @@
 #--------------------------------------------------------------------------------------------
 
 $ErrorActionPreference = "Stop"
+$InformationPreference = "Continue";
+
+$metadataUri = "http://metadata.google.internal/computeMetadata/v1/instance";
```

Comment:
Nit: variables should be upper case (at least that's the convention thruout the rest of the script)

Reply:
Will change the capitalization and align the diagnostic related variables.